src/Symfony/Component/Form/Tests/Extension/Core/Type/FormTypeTest.php

```diff
@@ -29,15 +29,11 @@ public function __construct($reference)
         $this->referenceCopy = $reference;
     }
 
-    // The returned object should be modified by reference without having
-    // to provide a setReference() method
```

Comment:
I would keep this comment and the next one (you can move them inside the method if that makes PHP-CS-Fixer happier :))